content/pt-br/docs/tasks/configure-pod-container/configure-runasusername.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Configurando `RunAsUserName` Para Pods e Contêineres Windows
+content_type: task
+weight: 20
+update_date: 2022-07-17
+origin_version: 1.24
+contributors: DonatoHorn
+reviewers:
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
+
+Esta página mostra como usar a configuração `runAsUserName` para Pods 
+e contêineres que serão executados em nós Windows. Isso é aproximadamente 
+equivalente à configuração `runAsUser` específica do Linux, permitindo a você 
+executar aplicativos em um contêiner com um nome de usuário diferente do padrão. 
+
+## {{% heading "prerequisites" %}}
+
+Você precisa ter um cluster Kubernetes, e a ferramenta de linha de comando Kubectl
+deve ser configurada para se comunicar com o seu cluster. Espera-se que o cluster
+tenha nós `worker Windows`, onde os Pods com contêineres executando as cargas de trabalho do Windows,
+serão agendados.
+
+<!-- steps -->
+
+## Defina o nome de usuário para um Pod
+
+Para especificar o nome de usuário com o qual executar os processos de contêiner do Pod, 
+inclua o campo `securityContext` ([SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)) 
+na especificação do Pod, e dentro dela, o campo `WindowsOptions` ([WindowsSecurityContextOptions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#windowssecuritycontextoptions-v1-core)) 
+contendo o campo `runAsUserName`.
+
+As opções de contexto de segurança do Windows que você especificar para um Pod, 
+se aplicam a todos os Contêineres do Pod, inclusive os de inicialização.
```

Comment:
If writing the translated term “Contêiner”, don't capitalize the first letter. It's OK to write “Container” if you are referring to the API definition within a Pod spec.

Reply:
Sorry, but I'm following the origin en docs.

Reply:
In English, “Container” is correct; the term in the API is `Container` and the local word is “container”.